src/platform/OpenThread/GenericNetworkCommissioningThreadDriver.cpp

```diff
@@ -70,7 +70,8 @@ CHIP_ERROR GenericThreadDriver::RevertConfiguration()
     return CHIP_NO_ERROR;
 }
 
-Status GenericThreadDriver::AddOrUpdateNetwork(ByteSpan operationalDataset)
+Status GenericThreadDriver::AddOrUpdateNetwork(ByteSpan operationalDataset, MutableCharSpan & outDebugText,
+                                               uint8_t & outNetworkIndex)
 {
     uint8_t extpanid[kSizeExtendedPanId];
```

Comment:
Zero `outDebutText` and `outNetworkIndex`?

Reply:
Done